src/main/java/com/targomo/client/Constants.java

```diff
@@ -114,4 +122,6 @@
     public static final String INTER_SERVICE_KEY                                        = "interServiceKey";
     public static final String TRAVEL_TIME_FACTORS										= "travelTimeFactors";
     public static final String GEO_JSON   												= "geojson";
+    public static final Integer MULTIGRAPH_DEFAULT_MIN_SOURCES_COUNT                    = 1;
+    public static final Boolean MULTIGRAPH_DEFAULT_IGNORE_OUTLIER                       = Boolean.FALSE;
```

Comment:
those defaults should probably not be here because they are internal defaults of the API core

also they should have different naming: `MULTIGRAPH_AGGREGATION_DEFAULT_MIN_SOURCES_COUNT` and `MULTIGRAPH_AGGREGATION_DEFAULT_IGNORE_OUTLIER`